src/Builder/Capacity/Lookup/All.php

```diff
@@ -71,9 +80,9 @@ public function getNode(): Node
                                 ),
                                 name: new Node\Identifier('queryParameters'),
                             ),
-                            $this->code !== null ? new Node\Arg(
-                                value: $this->code,
-                                name: new Node\Identifier('attributeCode'),
+                            $this->parameter !== null ? new Node\Arg(
+                                value: $this->parameter,
+                                name: new Node\Identifier($this->parameterName),
```

Comment:
Que se passe-t-il si la valeur de `$this->parameterName` est nulle ?

Reply:
les deux parametres sont maintenants indissociable.